packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -220,7 +220,20 @@ const processInlineImage = (props: ChatMessageContentProps): ProcessingInstructi
     };
     /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
     return (
-      <span onClick={handleOnClick} data-ui-id={node.attribs.id}>
+      <span
+        data-ui-id={node.attribs.id}
+        onClick={handleOnClick}
+        tabIndex={0}
+        role="button"
```

Comment:
Is this correct? Should we be treating this as a button? Won't it make accessibility incorrect?

Reply:
> Is this correct? Should we be treating this as a button? Won't it make accessibility incorrect?

This is the wrapper element of the img tag and it is acting as a button.
However, you brought up a good point that I should add aria-label to the img tag so the voice over can announce it more accurately.